README.md

```diff
@@ -28,33 +28,29 @@ Arch Linux or Ubuntu
 | `soaEditApi`         |                    | SOA-EDIT value when using the API. Defaults to `soaEdit`                                                                                            |
 | `dnssec`             | `false`            | Enable DNSSEC and NSEC3 for this zone                                                                                                               |
+| `defaultTtl`         | :heavy_check_mark: | TTL for all RRsets with no TTL given                                                                                                                |
 | `nsec3Iterations`    | `5`                | Amount of NSEC3 iterations                                                                                                                          |
 | `nsec3Salt`          | `dada`             | Salt to use when hashing for NSEC3                                                                                                                  |
 | `defaultNameservers` | :heavy_check_mark: | List of NS records (for `Master` and `Native` zones), or list of masters (for `Slave` zones). This is only used when creating the zone from scratch |
 | `metadata`           |                    | Dict with the domain metadata. Items that are present in the database, but not here are removed                                                     |
-| `rrsets`             | :heavy_check_mark: | List with all RRsets in this zone (see below)                                                                                                       |
+| `records`            | :heavy_check_mark: | List with all records in this zone (see below)                                                                                                      |
 
-### RRsets
-
-An RRset is identified by the name and the type.
-If both are the same, and the TTL or the records differ, the rrset is modified.
-Unknown RRsets are removed.
+### Records
 
-| Name      | Default/Required   | Description                                                       |
-|-----------|:------------------:|-------------------------------------------------------------------|
-| `name`    | :heavy_check_mark: | Name of this RRset                                                |
-| `type`    | :heavy_check_mark: | Record type of this RRset                                         |
-| `ttl`     | :heavy_check_mark: | TTL of this RRset                                                 |
-| `records` | :heavy_check_mark: | List of all records in this RRset. See below for more information |
+This role automatically sorts records of the same name and type into RRsets.
+Each record can either set a content (`c`) **or** can set a TTL which goes for the entire RRset (`t`).
 
-### Records
+Records are grouped into types which are grouped into names.
```

Comment:
grouped into names which are in turn grouped by type.

Reply:
![wrong](https://media.giphy.com/media/3oz8xLd9DJq2l2VFtu/giphy.gif)